resources/ext/ezbake.conf

```diff
@@ -16,6 +16,7 @@ ezbake: {
                # way to get rid of the hard-coded paths here, but I don't
                # know it.
                postinst: [
+                 "install --owner={{user}} --group={{user}} -d /opt/puppetlabs/server/data/puppetserver/jruby-gems",
```

Comment:
Yes, that's correct.  We didn't have this in here for master because the directory will be automatically created when `puppetserver gem install` runs, which is expected to be run as the puppet user or as root.

I pulled it in since it doesn't hurt and was explicitly there in some change between 1.0.2 and 1.0.8 and I didn't want it to vanish in the merge-up.